src/locales/en_US.ts

```diff
@@ -1307,6 +1307,9 @@ const strings = {
   fiat_plugin_sell_region_restricted: 'Region restricted from selling %s',
   fiat_plugin_sell_no_quote: 'Unable to get sell quote',
   fiat_plugin_fetching_assets: 'Fetching supported assets',
+  fiat_plugin_sell_cancelled: 'Sell order cancelled',
+  fiat_plugin_sell_failed_try_again: 'Sell order failed. Please try again.',
+  fiat_plugin_sell_failed_to_send_try_again: 'Failed to send funds for sell transaction. Please try again.',
```

Comment:
Wording suggestion:
'Funds transfer failed, unable to complete sell order. Your funds have not
been lost, please try again.'
'Crypto transfer failed, unable to complete sell order. Your funds have not
been lost. please try again.'

Trying to make it clear that:
1. This is talking about the same thing as the other two messages: `Sell order`
2. The funds simply failed to leave the wallet, they did not get sent and
   disappear. Describing crypto failing to leave the wallet, not that fiat
   failed to arrive.

Reply:
Let me pass this through Michael for feedback

Reply:
Main takeaways with Michael:

1. Don't tell the user to try again if the error message is describing a
   situation in which crypto has been sent already
2. Any other situation: `fiat_plugin_sell_failed_try_again: 'Sell order failed. Please try again.'` is sufficient to
   display, additional details on the specifics of the error can be hidden from the user